desk/app/heap.hoon

```diff
@@ -408,7 +408,7 @@
   |^
   =/  =perm:h
     :_  group.association
-    ?:(=(~ writers) ~ (silt (rap 3 %heap '-' (scot %p p.flag) '-' q.flag ~) ~))
+    ?:(=(~ writers) ~ (silt (rap 3 'imports/' (scot %p p.flag) '/' q.flag ~) ~))
```

Comment:
this one says imports the others say import

Reply:
Fixed